client/src/component/County/Dashboard/AuditBoardNumberSelector.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+
+import { NumericInput } from '@blueprintjs/core';
+
+import setNumberOfAuditBoards from 'corla/action/county/setNumberOfAuditBoards';
+
+
+/**
+ * Minimum number of audit boards that can participate.
+ */
+const MIN_AUDIT_BOARDS = 1;
+
+interface AuditBoardNumberSelectorProps {
+    auditBoardCount: number;
+    numberOfBallotsToAudit?: number;
+    isShown: boolean;
+    isEnabled: boolean;
+}
+
+interface AuditBoardNumberSelectorState {
+    auditBoardCount: number;
+    isEnabled: boolean;
+}
+
+class AuditBoardNumberSelector
+    extends React.Component<AuditBoardNumberSelectorProps,
+                            AuditBoardNumberSelectorState> {
+    constructor(props: AuditBoardNumberSelectorProps) {
+        super(props);
+
+        this.state = {
+          auditBoardCount: props.auditBoardCount,
+          isEnabled: props.isEnabled,
+        };
+
+        this.handleChangeAuditBoards = this.handleChangeAuditBoards.bind(this);
```

Comment:
I'm not immediately noticing what this circular-looking binding is about. Would you mind elaborating on this a bit?

Reply:
Thanks for the reminder. 🙉 🙈 🙊